api/types/src/transaction.rs

```diff
@@ -1032,6 +1032,7 @@ impl TryFrom<Script> for ScriptPayload {
 #[oai(one_of, discriminator_name = "type", rename_all = "snake_case")]
 pub enum MultisigTransactionPayload {
     EntryFunctionPayload(EntryFunctionPayload),
+    ScriptPayload(ScriptPayload),
```

Comment:
Why adding it here as well? We can just move to the new format,  that supports orderless as well?

Reply:
Orderless transactions can be significantly more expensive than non-orderless transactions, and I suspect that it is mostly unnecessary to be used with Multisig transactions.

SDK requires you to basically build some parts for the payload, so I suspect it may change the flows significantly depending on the language SDK used.